src/test/regress/checkinc.py

```diff
@@ -78,9 +78,10 @@
     'winsock.h':         [],
     'winsock2.h':        [],
     'ws2tcpip.h':        [],
-    'hdfs/hdfs.h': 		 [],
+    'hdfs/hdfs.h': 	 [],
     'quicklz1.h':	 [],
     'quicklz3.h':	 [],
+    'json-c/json.h':     [],
```

Comment:
Does `checkinc.py` apply only when executing `make install`? I think this fix is necessary but I don't have a Mac environment to test it.